schemas/account_paths.json

```diff
@@ -608,5 +608,96 @@
+                  "application/vnd.api+json; charset=utf-8": {
+                     "schema": {
+                        "type": "object",
+                        "title": "RepaymentInformationResponse",
+                        "properties": {
+                           "data": {
+                              "type": "object",
+                              "properties": {
+                                 "type": {
+                                    "type": "string",
+                                    "enum": [
+                                       "creditAccountRepaymentInformation"
+                                    ],
+                                    "default": "creditAccountRepaymentInformation"
+                                 },
+                                 "attributes": {
+                                    "type": "object",
+                                    "properties": {
+                                       "remainingAmountDue": {
+                                          "type": "integer"
+                                       },
+                                       "remainingAmountOverdue": {
+                                          "type": "integer"
+                                       },
+                                       "initiatedRepayments": {
+                                          "type": "integer"
+                                       },
+                                       "statementPeriodStart": {
+                                          "type": "string",
+                                          "format": "date"
+                                       },
+                                       "statementPeriodEnd": {
+                                          "type": "string",
+                                          "format": "date"
+                                       },
+                                       "nextRepaymentDueDate": {
+                                          "type": "string",
+                                          "format": "date"
+                                       }
+                                    },
+                                    "required": [
+                                       "remainingAmountDue",
+                                       "remainingAmountOverdue",
+                                       "initiatedRepayments",
+                                       "statementPeriodStart",
+                                       "statementPeriodEnd",
+                                       "nextRepaymentDueDate"
+                                    ],
+                                    "additionalProperties": false
+                                 }
+                              },
+                              "required": [
+                                 "type",
+                                 "attributes"
+                              ],
+                              "additionalProperties": false
```

Comment:
Let's extract this object into a separate file named `creditAccountRepaymentInformation`